sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -468,6 +469,20 @@ public ServiceBusSenderClientBuilder queueName(String queueName) {
             return this;
         }
 
+        /**
+         * Sets the name of the initial destination Service Bus queue to publish messages to.
+         *
+         * @param viaQueueName The initial destination of the message.
+         *
+         * @return The modified {@link ServiceBusSenderClientBuilder} object.
+         *
+         * @see <a href="https://docs.microsoft.com/azure/service-bus-messaging/service-bus-transactions#transfers-and-send-via">Send Via</a>
+         */
+        public ServiceBusSenderClientBuilder viaQueueName(String viaQueueName) {
```

Comment:
Is this only for queues? Does the entity type also apply to subscriptions? If only queues, should we have a runtime check it's not a topic?